server/dive_tasks/celeryconfig.py

```diff
@@ -44,12 +44,11 @@
 if broker_url is None:
     raise RuntimeError('CELERY_BROKER_URL must be set')
 
-broker_heartbeat = False
 worker_send_task_events = False
 # https://docs.celeryproject.org/en/stable/userguide/configuration.html#std-setting-worker_prefetch_multiplier
 worker_prefetch_multiplier = 1
 # https://docs.celeryproject.org/en/v4.4.6/userguide/configuration.html#broker-connection-timeout
-broker_connection_timeout = 12
+broker_connection_timeout = 6
```

Comment:
what are the downsides to this being more lenient?  From the documentation it is the worker connection timeout so I'm assuming that means the connection from the individual running the worker to the message broker.  Just thinking of edge cases, Hawaii, or satellite internet from a boat?

Reply:
User experience, really.  12 seconds is a long time.  If nothing happens for that long when you should be getting a job (or a message about having lost connection) you might get confused or frustrated.

That said, maybe it's not a big deal?